client/src/main/java/org/spine3/protobuf/Durations.java

```diff
@@ -304,4 +305,17 @@ public static boolean isNegative(Duration value) {
         final boolean isNegative = nanos < 0;
         return isNegative;
     }
+
+    /**
+     * Returns a duration between two timestamps.
```

Comment:
Have a look at [definition of Interval](http://www.threeten.org/threeten-extra/apidocs/org/threeten/extra/Interval.html) in Joda Time and [the question on a similar concept in Java 8](http://stackoverflow.com/questions/22886030/equivalent-to-jodatime-interval-in-java-8-date-and-time-api).

As long, as we talk about the distance between two timestamps, we need to add a new data type called `Interval` in our Values sub-project. Duration and Interval are quite different things. A Duration is, so to say, is ‘filled in’ with something, while an Interval, is ‘empty’.